sdk/tests/conformance/rendering/point-size-varyings.html

```diff
@@ -35,76 +33,68 @@
 <script src="../../js/webgl-test-utils.js"> </script>
 </head>
 <body>
-<canvas id="example" width="50" height="50">
-There is supposed to be an example drawing here, but it's not important.
-</canvas>
+<canvas id="c" width="64" height="64"></canvas>
 <div id="description"></div>
 <div id="console"></div>
 <script id="vshader" type="x-shader/x-vertex">
-precision highp float;
-uniform float isCloud; // never set
-attribute vec4 vPosition;
-varying vec2 vUV;
+attribute vec2 aPosition;
+
+varying vec2 vPos;
+
 void main()
 {
-    gl_Position = vPosition;
-    vUV = vPosition.xy * 2.0;
-    if (isCloud == 1.0) {
-      gl_PointSize = 1.0;
-    }
+    gl_Position = vec4(aPosition, 0, 1);
+    vPos = aPosition;
+
+    gl_PointSize = 16.0;
 }
 </script>
 
 <script id="fshader" type="x-shader/x-fragment">
-precision highp float;
-varying vec2 vUV;
-uniform float isCloud; // never set
+uniform mediump float defaultsToZero; // never set
+
+varying mediump vec2 vPos;
+
 void main()
 {
-    gl_FragColor = vec4(vUV.xy,0.0,1.0);
-    if (isCloud == 1.0) {
-      gl_FragColor = vec4(gl_PointCoord.xy,0.0,1.0);
+    gl_FragColor = vec4(vPos * -2.0, 0, 1);
+
+    if (defaultsToZero == 1.0) {
```

Comment:
even this is not needed.  Simply put gl_FragColor = vec4(gl_PointCoord.xy, 0, 1) before gl_FragColor = vec4(vPos * -2.0, 0, 1) is enough to trigger this bug.